spyder/plugins/completion/languageserver/confpage.py

```diff
@@ -240,16 +236,20 @@ def setup_page(self):
         self.code_style_check.toggled.connect(code_style_g_widget.setEnabled)
 
         # Code style layout
+        code_style_group = QGroupBox(_("Code style"))
         code_style_layout = QVBoxLayout()
         code_style_layout.addWidget(code_style_label)
         code_style_layout.addWidget(self.code_style_check)
         code_style_layout.addWidget(code_style_g_widget)
-        code_style_layout.addWidget(vertical_line_box)
+        code_style_group.setLayout(code_style_layout)
 
-        code_style_widget = QWidget()
-        code_style_widget.setLayout(code_style_layout)
+        # Maximum allowed line length layout
+        line_length_group = QGroupBox(_("Maximum allowed line length"))
```

Comment:
```suggestion
        line_length_group = QGroupBox(_("Line length"))
```

Just to not be repeat the text below.